test/test_helper.rb

```diff
@@ -126,8 +126,10 @@ class TestCase
     # Standard setup to run before every test.  Sets the locale, timezone,
     # and makes sure User doesn't think a user is logged in.
     def setup
-      I18n.locale = :en if I18n.locale != :en
-      Time.zone = "America/New_York"
+      # Disable cop; there's no block in which to limit the time zone change
+      I18n.locale = :en if I18n.locale != :en # rubocop:disable Rails/I18nLocaleAssignment
+      # Disable cop; there's no block in which to limit the time zone change
+      Time.zone = "America/New_York" # rubocop:disable Rails/TimeZoneAssignment
```

Comment:
Is there a way to use a `locate_with` or `use_zone` block in `setup`?

Reply:
I think we would need the equivalent of an "around_action" callback.  The structure now is it calls setup, the test, then teardown.  It seems like we would need an "around_test":

'def around_test
'  I18n.with_locale(:en) do
'    Time.with_zone(blah) do
'      yield # test
'    end
'  end
'end